### ISSUE TITLE
xdotool only types capital letters

### ISSUE THREAD

**Issue by Malabarba** [jordansissel/xdotool#74]
The same happens with the `key` command.
Here's an example terminal command with output.

    $ xdotool type --clearmodifiers smallletters
    SMALLLETTERS
    $ xdotool type --clearmodifiers BIGLETTERS
    BIGLETTERS

I'm on Arch Linux, and I'm pretty sure this is happening on an Ubuntu box at work (will check tomorrow).

Let me know if there's any more information you need.

**Comment by Malabarba** [comment]
Forgot to mention:
- Capslock is not activated.
- Activating capslock makes no difference.
- Holding down shift makes no difference.

**Comment by Malabarba** [comment]
Found the source. This is caused by using 

    setxkbmap -option caps:internal

when setting my kbd layout. Is this a known issue?

**Comment by jordansissel** [comment]
Sounds like a bug in xdotool. I have never used that xkbmap option so I am
not sure of the cause.

**Comment by Malabarba** [comment]
Yeah, I only enabled that because I found it in the manual, I've disabled it now and everything works.
It only took me a looong time to figure out because I didn't run into it through xdotools directly, but through keepass instead (and keepass linux client is pretty buggy already so I just thought it was their bug).

For the record, this option is supposed to make it so that shift inverts the capslock key.
